client/extensions/woocommerce/index.js

```diff
@@ -265,9 +266,19 @@ export default async function ( _, addReducer ) {
 	// Add pages that use the store navigation
 	getStorePages().forEach( function ( storePage ) {
 		if ( config.isEnabled( storePage.configKey ) ) {
-			addStorePage( storePage, ( context, next ) =>
-				createStoreNavigation( context, next, storePage )
-			);
+			// Store deprecation would redirect most store pages to dashboard
+			if ( isStoreRemoved && ! ( '/store/:site' === storePage.path ) ) {
+				page( storePage.path, addTracksContext, ( context ) => {
+					// Tracks MC stats by path /store/products/:site -> store-products-:site
+					const trackName = storePage.path.substr( 1 ).replaceAll( '/', '-' );
```

Comment:
This is not Tracks. This is MC stats. Please do not use the "tracks" terminology for this.

Reply:
In addition, from PCYsg-G3-p2:

> Always pass a static string for $statname and $bin or a variable that is verified against a whitelist of values in your code. 

It would be better to use a shorter bin name that doesn't have any special characters. That might be what's causing the 'undefined' results mentioned in the comments of this PR.

Reply:
Thanks for catching that! I've made the changes in https://github.com/Automattic/wp-calypso/pull/48249/commits/5fc92e6f6be7d14e429bb38d2720288323033bcd. 

I've opted to use routes' document title transformed to slug as the respective stat name. Some of the routes will use the same name (i.e: routes with filters/ids),  but I don't think that's a problem since we don't need that level of detail.

>That might be what's causing the 'undefined' results mentioned in the comments of this PR.

I've found out in slack that the issue is a problem that's happening in MC, and so far the symptom is not suggesting that the stat name is causing it. However, I do agree its better not use any special characters, I'll be sticking with just dashes which I confirm does work.